lib/docs/filters/padrino/entries.rb

```diff
@@ -0,0 +1,17 @@
+module Docs
+  class Padrino
+    class EntriesFilter < Docs::EntriesFilter
+      def get_name
+        node = at_css('h1')
+        result = node.content.strip
+        result << ' event' if type == 'Events'
+        result << '()' if node['class'].try(:include?, 'function')
+        result
+      end
+
+      def get_type
+        return 'Ruby'
```

Comment:
Hi @Thibaut methods and classes are now indexed (I checked some links in the original one and the generated one and I got same results). Now it's time to make the highlighting!